ada/tasks.py

```diff
@@ -22,3 +26,21 @@ def submit_whatsappid_to_rapidpro(whatsappid):
             flow=settings.ADA_PROTOTYPE_SURVEY_FLOW_ID,
             urns=[f"whatsapp:{whatsappid.lstrip('+')}"],
         )
+
+
+def submit_whatsappid_to_rapidpro_topup(whatsappid):
```

Comment:
Missing the `@app.task(` decorator for these two tasks

Reply:
I'm thinking the two `submit_whatsappid_to_rapidpro*` tasks could be better named, bit confusing.

`start_prototype_survey_flow` and `start_topup_flow` is a bit more descriptive of what they do.

Reply:
I agree.